src/Microsoft.TemplateEngine.Utils/CryptoRandom.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    public class CryptoRandom : Random
+    {
+        private RNGCryptoServiceProvider _rng = new RNGCryptoServiceProvider();
```

Comment:
[MSDN states](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rngcryptoserviceprovider?view=net-6.0) that `RNGCryptoServiceProvider` is obsolete.
It is probably better if we used [RandomNumberGenerator](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.randomnumbergenerator?view=netcore-3.1) as recommended.